hybrid/detailed_pv_plant.py

```diff
@@ -27,12 +27,15 @@ def __init__(self,
             'layout_config': optional dict, contains all keys for PVLayoutConfig dataclass. Required for layout modeling
         """
         system_model = Pvsam.default("FlatPlatePVSingleOwner")
-        financial_model = Singleowner.from_existing(system_model, "FlatPlatePVSingleOwner")
+
```

Comment:
Could you please add this option of pv_config to the docstring above?

Reply:
Updated, and removed parameter in docstring that will come next with the detailed layout implementation